source_pubsub/client.go

```diff
@@ -255,39 +261,67 @@ func (c *PubSubClient) Wait(ctx context.Context) error {
 	}
 }
 
+func (c *PubSubClient) ResetRetryCount() {
+	c.retryCount = c.maxRetries
+}
+
 func (c *PubSubClient) ReplayID() []byte {
 	return c.currReplayID
 }
 
+func (c *PubSubClient) retryAuth(ctx context.Context, retry bool) (error, bool) {
+	var err error
+	sdk.Logger(ctx).Info().Msgf("retry connection - retries %d ", c.retryCount)
```

Comment:
small nit on the log:
```suggestion
	sdk.Logger(ctx).Info().Msgf("retry connection - retries remaining %d ", c.retryCount)
```